content/en/synthetics/guide/identify_synthetics_bots.md

```diff
@@ -21,23 +21,25 @@ further_reading:
 
 ## Overview
 
-Some parts of your systems might not be available to robots without the right identification, or you might want to avoid collecting analytics from Datadog robots. You can use the below methods to detect Datadog Synthetic Monitoring robots.
+Some parts of your systems may not be available to robots without the appropriate identification. You may also want to avoid collecting analytics from Datadog robots. 
+
+Try a combination of the following methods across your Synthetic test configurations to detect Datadog's Synthetic Monitoring robots.
```

Comment:
```suggestion
You might need to try a combination of the following methods to detect Datadog's Synthetic Monitoring robots.
```

Reply:
We use "you may..." in the paragraph right before so using "you might" in this sentence may be a little repetitive IMO.

Reply:
I would be fine keeping this sentence as is!